File: web/src/app/models/office-hour.ts
import {Office} from './office';

export class OfficeHour {
  id: number;
  office: Office;
  beginTime: string;
  endTime: string;
  daytype: string;


  constructor() {
    this.id = 0;
    this.daytype = 'MONDAY';
    this.beginTime = '00:00';
    this.endTime = '00:00';
  }
}

File: mobile/app/src/main/java/wse18/ase/qse03/mobile/model/OfficeHour.kt
package wse18.ase.qse03.mobile.model

import java.io.Serializable

class OfficeHour (
    val id: Long,
    var beginTime: String,
    var endTime: String,
    var daytype: String
): Serializable {
    constructor() : this(-1, "", "", "")
    override fun toString(): String = daytype.substring(0,1) + daytype.substring(1,3).toLowerCase() + ": " + beginTime + " - " + endTime
}